src/components/MessageInput/InputButtons.tsx

```diff
@@ -57,6 +57,8 @@ export type InputButtonsWithContextProps<
   closeAttachmentPicker?: () => void;
   openAttachmentPicker?: () => void;
   openCommandsPicker?: () => void;
+  openFilePicker?: () => void;
+  openMentionsPicker?: () => void;
```

Comment:
hmm I don't love adding props that aren't actually used within our component and are magically there to provide functionality to an example app (I assume this is for the slack clone). there may be a more creative way we can create and pull these (or their functionalities) from context instead of creating functions in our SDK which are doing nothing by default

Reply:
> I don't love adding props that aren't actually used within our component 
I totally agree on this one. But this one seems like a good thing to have. Giving access to all "pickers" as simplified functions to end user. I would insist on keeping this one

Reply:
Raised a PR here - https://github.com/GetStream/stream-chat-react-native/pull/576/files